### ISSUE TITLE
Preprocessing with sklearn: a complete and comprehensive guide

### ISSUE THREAD

**Issue by RobertGarciadSilva** [khuyentran1401/Web-Scrapping-Wikipedia#1]
## TL;DR
A useful guide to preprocessing data using sklearn.

### Article link
https://towardsdatascience.com/preprocessing-with-sklearn-a-complete-and-comprehensive-guide-670cb98fcfb9

## Key Takeaways
The following subjects will be handled:

- Missing values
- Polynomial features
- Categorical features
- Numerical features
- Custom transformations
- Feature scaling
- Normalization
